colibri/doc/sphinx/source/tutorials/running_fits/bayesian/ultranest.rst

```diff
@@ -0,0 +1,167 @@
+
+.. _ultranest_fit:
+
+===========================
+Bayesian Fit with UltraNest
+===========================
+
+We will first look at an example runcard to run a Bayesian fit with the Les Houches
+parametrisation model (see :ref:`this tutorial <in_les_houches>` for details
+on the Les Houches model and how to build it).
+
+Then we will look at the command to execute the runcard. 
+
+Runcard
+-------
+
+.. code-block:: bash
+
+    meta: 'An example fit using Colibri, reduced DIS dataset.'
+
+    #######################
+    # Data and theory specs
+    #######################
```

Comment:
Hi @vschutze-alt thanks for adding this structure, it reads much better now. I would like to also see some discussion on the ns_settings, as this should be the main focus of this section.